components/modules/NewReleases/index.tsx

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable @next/next/no-img-element */
+import React, { useState } from "react";
+import { songs } from "../utils/dummy";
+import { VDots, VPause, VPlay, VStar, VWave } from "../__modules__/_Vectors";
+import liveBg from "../../../public/livebg.jpg";
+import AudioPlayer from "../utils/config/audioPlayer";
+import equalizer from "../static/images/equalizer.gif";
+
+interface TrackItems{
+  [index: number]:{title: string;
```

Comment:
Why this notation on index ?
```suggestion
  [index: number]:{title: string;}
```